01/sonar_sweep.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+
+def window_total(measurements, window):
+  """Gets the total sum of measurements across an arbitrary range length (window)
+
+        Parameters:
+          measurements (list): List of measurements to take values from
+          window       (int) : Size of range to sum over
+
+        Returns:
+          total (int): Total of the sum over the window 
+  """
+
+  total = 0
+
+  for i in range(window):
+    total += measurements[i]
+
+  return total
+
+
+def find_increase_count(measurements, window = 1):
+  """For a given set of measurements and for a give window size, find how many times there's an increase in the sum of consecutive measurements 
+
+        Parameters:
+          measurements (list): List of measurements to take values from
+          window       (int) : Size of range to sum over
+
+        Returns:
+          higher_count (int): How many times there's an increase in the sum of consecutive measurements 
+  """
+
+  previous_measurement = None
+  higher_count = 0
+  i = 0
+  for measurement in measurements[:len(measurements) - window + 1]:
+    total = window_total(measurements[i : i + window], window)
+
+    if previous_measurement is not None and total > previous_measurement:
```

Comment:
You can just do 'if previous_measurement' instead of 'if previous_measurement is not None'